### PR TITLE
feat: new lacework_alert_channel_webhook resource

### PR DESCRIPTION
**Pull request by dmurray-lacework**
Add new resource `lacework_alert_channel_webhook`

### CODE REVIEW

File: website/docs/r/alert_channel_webhook.html.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+subcategory: "Alert Channels"
+layout: "lacework"
+page_title: "Lacework: lacework_alert_channel_webhook"
+description: |-
+  Create and manage Webhook Alert Channel integrations
+---
+
+# lacework\_alert\_channel\_webhook
+
+Configure Lacework to forward alerts to a 3rd party Webhook.
```
Comment (afiune) [inline website/docs/r/alert_channel_webhook.html.markdown:11]:
```suggestion
You can use this resource to create a custom webhook that receives Lacework alert notifications from a Lacework alert channel and forwards those alerts to a 3rd party application.
```

File: website/docs/r/alert_channel_webhook.html.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+-> **Note:**  If the third-party that is receiving the HTTP POST request requires an API token, enter the API Token as part of the URL eg. https://webhook.com?api-token=123
```
Comment (afiune) [inline website/docs/r/alert_channel_webhook.html.markdown:13]:
```suggestion
-> **Note:**  If the third-party application receiving the HTTP POST request requires an API token, enter the API token as part of the URL eg. https://webhook.com?api-token=123
```

File: website/docs/r/alert_channel_webhook.html.markdown [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+layout: "lacework"
+page_title: "Lacework: lacework_alert_channel_webhook"
+description: |-
+  Create and manage Webhook Alert Channel integrations
+---
+
+# lacework\_alert\_channel\_webhook
+
+You can use this resource to create a custom webhook that receives Lacework alert notifications from a Lacework alert channel and forwards those alerts to a third-party application.
+
```
Comment (afiune) [inline website/docs/r/alert_channel_webhook.html.markdown:12]: here

File: website/docs/r/alert_channel_webhook.html.markdown [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+-> **Note:**  If the third-party application receiving the HTTP POST request requires an API token, enter the API token as part of the URL eg. https://webhook.com?api-token=123
+
+To find more information about the payload view the lacework support documentation
```
Comment (afiune) [inline website/docs/r/alert_channel_webhook.html.markdown:15]: Lacework needs to be written capitalized.

File: website/docs/r/alert_channel_webhook.html.markdown [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+https://support.lacework.com/hc/en-us/articles/360034367393-Webhook
```
Comment (afiune) [inline website/docs/r/alert_channel_webhook.html.markdown:16]:
```suggestion
To find more information about the alert payload, see the [Lacework support documentation](https://support.lacework.com/hc/en-us/articles/360034367393-Webhook).
```

File: website/docs/r/alert_channel_webhook.html.markdown (hunk 4 above)
Comment (dmurray-lacework) [reply website/docs/r/alert_channel_webhook.html.markdown]: Ah, missed that. Thanks!

**Review by afiune — APPROVED** [review]
tenor-242799352

**Pull request merged**